packages/components/select/__tests__/select.test.ts

```diff
@@ -2480,111 +2480,113 @@ describe('Select', () => {
       const formItem = wrapper.find('[data-test-ref="item"]')
       expect(formItem.attributes().role).toBe('group')
     })
-    // fix: 8544
-    it('When props are changed, label can be displayed correctly after selecting operation', async () => {
```

Comment:
like this test case should not be contained in the group `form item accessibility integration`

Reply:
I see. Thanks!